server/address: "192.168.10.250"
server/port: 6666

master: "0000000000002000"
num_anchors: 3

anchors/address: [
  "0000000000002000",
  "0000000000002002",
  "0000000000002001",
]
anchors/sn: [
  "05",
  "49",
  "17",
]
anchors/position: [
  #x,y,z
  0.29, 0.165, 0.135,
  0.29, -0.165, 0.135,
  -0.37, 0.0, 0.135,
]

target/tags: ["5a84", "5722", "c4b5"]
target/anchor: "da12"